homeassistant/components/websocket_api/messages.py

```diff
@@ -65,12 +65,31 @@ def construct_result_message(iden: int, payload: str) -> str:
     return f'{{"id":{iden},"type":"result","success":true,"result":{payload}}}'
 
 
-def error_message(iden: int | None, code: str, message: str) -> dict[str, Any]:
+def error_message(
+    iden: int | None,
+    code: str,
+    message: str,
+    translation_key: str | None = None,
+    translation_domain: str | None = None,
+    translation_placeholders: dict[str, Any] | None = None,
+) -> dict[str, Any]:
     """Return an error result message."""
+    error_payload: dict[str, Any] = {
+        "code": code,
+        "message": message,
+    }
+    # In case a ServiceNotFound error is raised
+    # we use 'websocket_api' as default translation_domain.
```

Comment:
This comment seems to be in the wrong place.

Reply:
That was changed to 'homeassistant'. But it must be set by the caller. So the comment is stale.

Reply:
Removed the comment